### PR TITLE
Math-Magician

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Components for the project

- create the components
- add it to App.js
- display App in index.js
- add style to it

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <button type="button" className="btn-gray">2</button>
+            <button type="button" className="btn-gray">3</button>
+            <button type="button" className="btn-orang">+</button>
+          </div>
+          <div className="rows">
+            <button type="button" className="btn-gray">AC</button>
+            <button type="button" className="btn-gray">+/-</button>
+            <button type="button" className="btn-gray">%</button>
+            <button type="button" className="btn-orang">+</button>
+          </div>
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] The symbol on line 19 should be the divide operation sign, not plus. Kindly fix that using the correct sign (&divide;) or the HTML code: `&divide;`

- [ ] The code from lines 41 to 44 is a repetition of lines 16 to 19. Kindly remove them.

  Your layout | Expected layout
  :---:|:---:
  dkddii | rurt

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AhmadiGu,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return (
+      <div className="container">
+        <div className="calculate-place">0</div>
+        <div className="functions-place">
+
+          <div className="rows">
+            <button type="button" className="btn-gray">AC</button>
+            <button type="button" className="btn-gray">+/-</button>
+            <button type="button" className="btn-gray">%</button>
+            <button type="button" className="btn-orang">+</button>
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [ ] As mentioned by the previous reviewer, Please change this add sign (+) into divide sign (÷) to match the given design.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @AhmadiGu,

Nice job making the required changes. Unfortunately, you missed some of them but you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @AhmadiGu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Calculator UI matches the given design :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**